Assets/MixedRealityToolkit.SDK/Inspectors/UX/Pointers/LinePointerInspector.cs

```diff
@@ -18,6 +18,7 @@ public class LinePointerInspector : BaseControllerPointerInspector
         private SerializedProperty lineRenderers;
 
         private bool linePointerFoldout = true;
+        private const int maxRecommendedLinecastResolution = 25;
```

Comment:
The [Tooltip on LinePointer](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3479/files#diff-42582bc1e2308bec1f932a22d56c0488R20) recommends 20 as the max for perf reasons. Should we standardize on 20 or 25 between here and there?